### DISCUSSION TITLE
HTML Extra report issues

### DISCUSSION THREAD

**Discussion by nvar234** [DannyDainton/newman-reporter-htmlextra#396] [Q&A]
hello I am trying to generate html extra report for the postman result summary and in the report the failure messages are not readable. It shows something like this - 
![Screenshot (142)](https://user-images.githubusercontent.com/115650671/195389538-bdeb83ef-27f4-4222-9515-b44fde048280.png)

**Comment by DannyDainton** [comment]
It's pushing the same error message as you would see in the UI or on the CLI reporter. The HTML report doesn't do something different with the output.

To me looks like you have variables in your URL which are not getting resolved at runtime. 
- Are these in an environment that you haven't selected for the run? 
- Are they global and you haven't attached that to the newman command? 
- Are they somewhere else?

Given that you're showing me a tiny cropped image and no other details about the overall newman run, I'm not sure I can really help here. 🤔

**Comment by nvar234** [comment]
In the UI I am seeing the proper error messages telling me exactly what's failing and why its failing whats expected and what's coming in response which is causing it to fail. Yes I do have environment variable and some global variables in my postman scripts which get resolved during runtime.
One of the failures how it shows up in UI ---
Transaction Result - Expected : Declined | Actual : Approved  |   AssertionError: expected 'Approved' to deeply equal 'Declined'.
But in the Html report its not very meaningful.

**Reply by DannyDainton** [reply]
If those details are not returned in the Newman Summary object, then they won't be in the HTML report.

I can't show something that's not there, every reporter would show the same thing as it's using the same underlying data from Newman.

What do you see in the console if you use the CLI reporter?
